config/rbac/role.yaml

```diff
@@ -92,6 +92,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - apps
+  resources:
+  - daemonsets
+  verbs:
+  - get
```

Comment:
no need `delete` or `patch`?

Reply:
So, I saw these errors for missing permissions for daemonsets and deploymentconfigs and added them based on the errors. These resource dependencies are not coming from model-registry though. So, it looks like they were missing in the original incubation branch.

Reply:
ok, I see. I thought modelreg requires such resources back.
I did the cleanup for `daemonsets` and `deploymentconfigs` a while a go, after confirmed with workbench team these are not in use any more.
do you still have the error logs where they showed up? I would like to take a look if this is a bug from my original PR or not.

Reply:
The errors I got on Dec 19 were:
```
W1220 05:01:48.998743 1 reflector.go:424] go/pkg/mod/k8s.io/client-go@v0.26.0/tools/cache/reflector.go:169: failed to list *v1.DeploymentConfig: deploymentconfigs.apps.openshift.io is forbidden: User "system:serviceaccount:opendatahub-operator-system:opendatahub-operator-controller-manager" cannot list resource "deploymentconfigs" in API group "apps.openshift.io" at the cluster scope
E1220 05:01:48.998767 1 reflector.go:140] go/pkg/mod/k8s.io/client-go@v0.26.0/tools/cache/reflector.go:169: Failed to watch *v1.DeploymentConfig: failed to list *v1.DeploymentConfig: deploymentconfigs.apps.openshift.io is forbidden: User "system:serviceaccount:opendatahub-operator-system:opendatahub-operator-controller-manager" cannot list resource "deploymentconfigs" in API group "apps.openshift.io" at the cluster scope
W1220 05:01:49.005990 1 reflector.go:424] go/pkg/mod/k8s.io/client-go@v0.26.0/tools/cache/reflector.go:169: failed to list *v1.DaemonSet: daemonsets.apps is forbidden: User "system:serviceaccount:opendatahub-operator-system:opendatahub-operator-controller-manager" cannot list resource "daemonsets" in API group "apps" at the cluster scope
E1220 05:01:49.006215 1 reflector.go:140] go/pkg/mod/k8s.io/client-go@v0.26.0/tools/cache/reflector.go:169: Failed to watch *v1.DaemonSet: failed to list *v1.DaemonSet: daemonsets.apps is forbidden: User "system:serviceaccount:opendatahub-operator-system:opendatahub-operator-controller-manager" cannot list resource "daemonsets" in API group "apps" at the cluster scope
```

Maybe you fixed that after I added it to this PR.